src/shaderlib/picking/shadertools-picking.md

```diff
@@ -0,0 +1,77 @@
+# picking (Shader Module)
+
+Provides support for color-coding-based picking and highlighting. In particular, supports picking a specific instance in an instanced draw call and highlighting an instance based on its picking color.
+
+Color based picking lets the application draw a primitive with a color that can later be used to index this specific primitive.
+
+Highlighting allows application to specify picking color of instance that need to be highlighted and the highlight color to be used.
```

Comment:
**DOC CHANGES** Add: to specify *a* picking color corresponding to an...

Change `instance`=>`object`? (shader is not always instanced)